README.md

```diff
@@ -1,50 +1,34 @@
-# PDS Template Repository for Python
+# Registry Sweepers
 
-This is the template repository for PDS's Python projects.
+This package provides supplementary metadata generation for registry documents, which are necessary for correct function of registry-api, and for common user queries. Execution is idempotent and should be scheduled on a recurring basis.
```

Comment:
I recommend chaning "which are necessary for correct function of registry-api" to "which are necessary for the correct functioning of the registry-api"

Reply:
Yeah, not sure what I was thinking with that phrasing - have amended.